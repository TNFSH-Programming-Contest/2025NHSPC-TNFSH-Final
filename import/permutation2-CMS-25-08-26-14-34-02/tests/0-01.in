3 2
20805 20804 20806
1 2
