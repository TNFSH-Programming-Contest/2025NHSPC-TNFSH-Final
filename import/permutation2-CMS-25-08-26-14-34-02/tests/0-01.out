BwX79QYzyAbxnmpqVLfwPV2Niz9fYyb5
2 3
0
